kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -336,7 +335,7 @@ class BatchJobSubmission(
     }
   }
 
-  override def close(): Unit = withLockRequired {
+  override def close(): Unit = withLockRequired(withClosingOperationLog {
```

Comment:
nit: `(` => `{`

Reply:
This was to prevent changing the idents of following lines.

Reply:
not a big deal, Git tools like GitHub could ignore space changes during diff view.
`()` is also fine if you insist.

Reply:
Thanks, let me keep it this time. The blame history is more valuable here for me.